src/dictionaries/EcKeyAlgorithm.js

```diff
@@ -9,7 +9,7 @@ const KeyAlgorithm = require('./KeyAlgorithm')
 const mapping = [
   { namedCurve: 'P-256', name: 'prime256v1', alg: 'ES256', hash: 'sha256' },
   { namedCurve: 'P-384', name: 'secp384r1', alg: 'ES384', hash: 'sha384' },
-  { namedCurve: 'P-512', name: 'secp521r1', alg: 'ES512', hash: 'sha512' },
+  { namedCurve: 'P-521', name: 'secp521r1', alg: 'ES521', hash: 'sha512' },
```

Comment:
Double checked this and while `P-521` is the correct curve name, the correct JWA string is `ES512`.

Reply:
You're totally right, missed this